pkg/cryptoinfo/identify.go

```diff
@@ -3,31 +3,29 @@
 // with dataflatten, and may eventually it may replace pkg/keyidentifier
 package cryptoinfo
 
-import (
-	"bytes"
-)
+type identifierSigfunc func(data []byte, password string) (results []*KeyInfo, err error)
```

Comment:
What do you think about naming this something like `keyIdentifier`/`cryptoIdentifier`/`keyinfoIdentifier`?

including `func` in the type name feels a bit redundant.

Reply:
I don't think it's worth it. This is an internal type, used only to declare that array. I think the _correct_ way is to use structs and interfaces, and think harder about package boundaries. But I also think that's premature right now. So this was easy enough. 

I guess "Sig" and "func" are kinda redundant. I could change it to `identifierSignature` or something